### ISSUE TITLE
Paragraph missing link

### ISSUE THREAD

**Issue by surajronline** [surajr1711/github_collaboration#1]
paragraph is missing link to purchase product. please add link.

**Comment by surajr1711** [comment]
yes youre right. the product wasnt live and setup wasnt completed so link was not included. marketing sent me the live link this morning.
Link will be fixed today.
